operator/v1/types_openshiftcontrollermanager.go

```diff
@@ -9,6 +9,7 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // OpenShiftControllerManager provides information to configure an operator to manage openshift-controller-manager.
+// +openshift:compatibility-gen:level=1
```

Comment:
I think i want to start these not exposed.  the API compatibility doc indicates

> It is common practice for Operators to use Custom Resource Definitions (CRDs)  internally to accomplish a task.  These objects are not meant for use by actors external to the operator and are intended to be “hidden”.  If any CRD is not meant for use by actors external to the operator, the operators.operatorframework.io/internal-objects annotation in the Operators ClusterServiceVersion (CSV) should be specified and that signals the corresponding resource is internal use only.

https://docs.google.com/document/d/1dkfqbGfi__Cs9UZMZrv1zGaJwXoHxO84YQkMlvrfAo4/edit

let's claim that.  This can't merge until 4.8 anyway.  I think we can do this by allowing it to be exposed=false.  Perhaps a better phrasing matches the compatibility doc. `internal=true`? That defaults better.

If you can remove this package from the initial pass I think we're close enough to tag and tidy.